.github/workflows/integration-tests.yml

```diff
@@ -17,9 +17,9 @@ jobs:
         id: set-matrix
         run: |
           if [ x"${{ github.repository }}" == x"openai/triton" ]; then
-            echo '::set-output name=matrix::[["self-hosted", "A10"], ["self-hosted", "V100"], "macos-10.15"]'
+            echo '::set-output name=matrix::[["self-hosted", "A10"], ["self-hosted", "V100"], "macos-10.15"]' >> $GITHUB_OUTPUT
```

Comment:
Here didn't update the macOS version as I was not sure if it was important to use this particular version